app/client/views/application/applicationCtrl.js

```diff
@@ -77,6 +76,7 @@ angular.module('reg')
               type: "success",
               confirmButtonColor: "#e76482"
             }, function(){
+              $rootScope.currentUser = data;
```

Comment:
This object is used [here](https://github.com/Fenixhack/alfarroba/blob/master/app/client/views/sidebar/sidebarCtrl.js#L13). One option is to update the object here and let Angular's binding and digest cycle take care of the view update.